### ISSUE TITLE
2.4.1 Kotlin Multiplaform. Plurals shows huge numbers from nowhere.

### ISSUE THREAD

**Issue by gswierczynski** [PocketByte/LocoLaser#24]
Since I switched from 1.0.4/2.2.1 to 2.4.1/2.4.1 in my Kotlin Multiplatform project the plurals on iOS seem to show huge numbers totally unrelated to the values actually passed to Repository method.

I double checked with native iOS way of displaying plurals and it is displaying correctly - so my dictionary setup is ok.

Is there any specific information that I could provide to help pinpoint what is the issue?

By huge numbers I mean tens and hundreds of thousands when the value passed is one digit.

Also the value doesn't seem to have impact on the value because larger value not necessarily produce larger huge number.

This is how I initialise the repository for iOS:
```
import ru.pocketbyte.locolaser.kmpp.IosStringRepository
import ru.pocketbyte.locolaser.kmpp.StringRepository

actual object Repository {
    actual val str: StringRepository = IosStringRepository()
    actual val non: NonTranslatableStrings = NonTranslatableStrings
}
```

Also passing the same one digit value twice gives two different huge numbers so it is totally not based on the value passed in.

**Comment by gswierczynski** [comment]
<img width="1019" alt="Screenshot 2024-07-02 at 21 18 03" src="https://github.com/PocketByte/LocoLaser/assets/3669529/ee31a641-dd7b-4b9e-86dd-b58703f894de">

**Comment by KamiSempai** [comment]
Fixed in version 2.4.2
